packages/react/src/patterns/sub-patterns/ContentBlock/ContentBlock.js

```diff
@@ -55,10 +56,58 @@ const ContentBlock = ({ heading, copy, children, customClassName, cta }) => {
         {children}
       </div>
       {cta && _renderCTA(cta)}
+    </>
+  );
+
+  const title = (
+    <div>
+      {heading && (
+        <h2
+          data-autoid={`${stablePrefix}--content-block__heading`}
+          className={`${prefix}--content-block__heading`}>
+          {heading}
+        </h2>
+      )}
+    </div>
+  );
+
+  return (
+    <div
+      data-autoid={`${stablePrefix}--content-block`}
+      className={cx(`${prefix}--content-block`, customClassName)}>
+      {linkList
+        ? _layoutWrap(
+            <>
+              {title}
+              <div></div>
+            </>
+          )
+        : title}
+      {linkList
+        ? _layoutWrap(
+            <>
+              <div>{content}</div>
+              <div>
+                <LinkList {...linkList} />
```

Comment:
Should link list be built directly into content block, or should it just allow an adopter to pass in link list and other items?

Reply:
ok, let me try that out